src/conversion/conversion.js

```diff
@@ -0,0 +1,40 @@
+// copy-paste from https://gist.github.com/letanure/4a81adfbda16b52d25ed
+
+angular.module('dd.ui.conversion', [])
+    .filter('distance', function () {
```

Comment:
Does it work with strictdi mode? https://docs.angularjs.org/error/$injector/strictdi